bom.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: bom
+  version: 0.4.1
+  epoch: 0
+  description: A utility to generate SPDX-compliant Bill of Materials manifests
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+
+environment:
+  contents:
+    packages:
+      - git
```

Comment:
Are we able to remove this section now that https://github.com/chainguard-dev/melange/pull/239 has merged?

Reply:
oh cool, let's remove it! Thanks for keeping me informed about that @luhring 👋